lib/Wikia/src/Service/User/Attributes/AttributeKeyValueService.php

```diff
@@ -87,15 +87,15 @@ public function delete( $userId, Attribute $attribute ) {
 			$ret = $this->persistenceAdapter->deleteAttribute( $userId, $attribute );
 			return $ret;
 		} catch ( \Exception $e ) {
-			$this->logError( $userId, $e );
+			$this->logError( $userId, $e, "USER_ATTRIBUTE error deleting from service" );
 			return false;
 		}
 	}
 
-	private function logError( $userId, \Exception $e ) {
-		$this->error( $e->getMessage(), [
+	private function logError( $userId, \Exception $e, $msg ) {
+		$this->error( $msg , [
 			'user' => $userId,
-			'exceptionType' => get_class( $e ),
+			'exception' => $e
```

Comment:
By passing the entire exception, we get access to the stack trace. As in [these logs](https://kibana.wikia-inc.com/index.html#dashboard/temp/AVOwK8vewuxBx3joFaRw).